src/styles/NavbarStyles/index.less

```diff
@@ -1,123 +1,188 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Staatliches&display=swap');
 
-.mainNav {
-  height: 12vh;
-  width: 100%;
-  background: #ffad00;
+.navbar__mainNav {
   display: flex;
+  background-color: #feae28;
+  min-height: 98px;
+  padding: 0 0.6rem;
+  width: 100%;
+  justify-content: space-between;
   align-items: center;
-  justify-content: space-evenly;
-  margin-bottom: 0%;
-  position: sticky;
-  top: 0;
-  z-index: 3;
-}
-
-.navLogo {
-  width: 20%;
-  display: flex;
-  align-items: center;
-}
-.navTitle {
-  margin: 0 0 0 0.8rem;
-  font-size: 2.2rem;
-  letter-spacing: 1px;
-  align-items: center;
-  font-weight: 500;
-  color: rgb(58, 9, 42);
-  font-family: 'Staatliches', 'cursive';
+  font-family: 'Montserrat';
```

Comment:
The less variable `@text__font-family` from our theme file could be in use here.